x-pack/plugins/lens/public/indexpattern_datasource/index.ts

```diff
@@ -35,6 +36,7 @@ export class IndexPatternDatasource {
     editorFrame.registerDatasource(async () => {
       const { getIndexPatternDatasource, renameColumns } = await import('../async_services');
```

Comment:
I like the new setup of making this an implementation detail of the datasource - can we move the `formatColumn` import into the `async_services` so it's loaded async?